odoo/addons/base/tests/test_reports.py

```diff
@@ -12,7 +12,12 @@
 @odoo.tests.tagged('post_install', '-at_install', 'post_install_l10n')
 class TestReports(odoo.tests.TransactionCase):
     def test_reports(self):
-        domain = [('report_type', 'like', 'qweb')]
+        blacklisted_reports = [
+            'account.report_original_vendor_bill',
+            'account.report_invoice_with_payments',
+            'account.report_invoice',
+        ]
+        domain = [('report_type', 'like', 'qweb'), ('report_name', 'not in', blacklisted_reports)]
```

Comment:
i'd rather have a domain to filter account_move_line per domain filtering the reports, in order to keep everything as much tested as before


something like
```python
domains = {
    []: [('report_type', 'like', 'qweb'), ('report_name', 'not in', blacklisted_reports)],
    [('move_id.move_type', 'in', ['out_invoice', 'out_refund', 'in_invoice', 'in_refund'])]:  [('report_type', 'like', 'qweb'), ('report_name', 'in', blacklisted_reports)],
}
for aml_domain, report_domain in domains.items():
    blablabla
```